diff --git a/node_modules/react-native/ReactCommon/hermes/executor/HermesExecutorFactory.cpp b/node_modules/react-native/ReactCommon/hermes/executor/HermesExecutorFactory.cpp
index 0b72b72..094e142 100644
--- a/node_modules/react-native/ReactCommon/hermes/executor/HermesExecutorFactory.cpp
+++ b/node_modules/react-native/ReactCommon/hermes/executor/HermesExecutorFactory.cpp
@@ -12,6 +12,8 @@
 #include <hermes/hermes.h>
 #include <jsi/decorator.h>
 #include <jsinspector-modern/InspectorFlags.h>
+#include <atomic>
+#include <thread>
 
 #include <hermes/inspector-modern/chrome/HermesRuntimeTargetDelegate.h>
 #include <hermes/inspector-modern/chrome/Registration.h>
